torchvision/datasets/utils.py

```diff
@@ -127,3 +127,58 @@ def list_files(root, suffix, prefix=False):
         files = [os.path.join(root, d) for d in files]
 
     return files
+
+
+def download_file_from_google_drive(id, root, filename=None, md5=None):
+    """Download a Google Drive file from  and place it in root.
+
+    Args:
+        url (str): URL to download file from
```

Comment:
the doc mention `url`, while the code is written `id` (which is by the way a Python method which you are shadowing in this function).  Could you make it consistent, by either fixing the doc or changing the name in the function argument? I'd prefer to change from `id` to something else